test/api-request-panel.test.js

```diff
@@ -2,7 +2,7 @@ import { fixture, assert } from '@open-wc/testing';
 import * as sinon from 'sinon/pkg/sinon-esm.js';
 import '../api-request-panel.js';
 
-describe('<api-request-panel>', function() {
+describe('<api-request-panel>', function () {
```

Comment:
We need to agree on common linting rules :)

Reply:
Agreed. I'll undo this